src/templates/app/new_account/welcome_page.jsx

```diff
@@ -1,46 +1,72 @@
-import React             from 'react';
+import React from 'react';
 import { SeparatorLine } from '../../_common/components/separator_line.jsx';
 
-const WelcomePage = () => (
-    <div id='welcome_container' className='center-text static_full invisible'>
-        <span className='account-created' />
+const WelcomePage = () => {
 
-        <SeparatorLine className='gr-padding-5' invisible />
+    const dropdown = 'images/pages/welcome/ddown.svg';
+    const options = [
+        {
+            icon_list: ['images/pages/welcome/cfd.svg'],
+            title    : it.L('CFD'),
+            desc     : it.L('<strong>Maximise returns</strong> by risking more.'),
```

Comment:
<strong> should be a placeholder so translators don't change it

Reply:
done